### ISSUE TITLE
Beacon Unable to Login to Streamline Servers via FTP

### ISSUE THREAD

**Issue by Moobien** [thommcgrath/Beacon#178]
I inputted all the information correctly for the FTP connection but it fails on me.  Not sure if its my end, your end or Streamline's end.  FileZilla and EasyFTP can both connect with no issues.

https://imgur.com/a/IjrC8Wr

**Comment by thommcgrath** [comment]
I will have to find some time to rent a server from them to figure out what the issue might be. Until then, I can’t really speculate.

**Comment by Moobien** [comment]
this is something they said yesterday in the ticket I opened with them. "This might be some sort of permissions related issue where Beacon is trying to do something that is not being allowed on our end, though I am not 100% sure."

**Comment by thommcgrath** [comment]
Yeah that’s what I mean. I’ll need to see what happens on their server to know more. Beacon only needs to list, download, and upload files. Normal stuff, but there’s too many possibilities to speculate.

**Comment by Moobien** [comment]
If need be I could give you access to mine

**Comment by thommcgrath** [comment]
I may take you up on that, but for now, it’s an issue of time.

**Comment by Moobien** [comment]
No worries
